docs/cli-usage.md

```diff
@@ -0,0 +1,141 @@
+# CLI Usage
+
+The Cloud SQL Proxy takes a few arguments to configure what instances to connect
+to and connection behavior.
+
+## Supported Flags
+
+* `-fuse`
+
+Requires access to `/dev/fuse` as well as the `fusermount` binary. An optional
+`-fuse_tmp` flag can specify where to place temporary files. The directory
+indicated by `-dir` is mounted.
+
+* `-instances="project1:region:instance1,project3:region:instance1"`
+
+A comma-separated list of instances to open inside `-dir`. Also supports
+exposing a tcp port and renaming the default Unix Domain Sockets; see examples
+below.  Same list can be provided via INSTANCES environment variable, in case
+when both are provided - proxy will use command line flag.
+
+* `-instances_metadata=metadata_key`
+
+Usable on [GCE](https://cloud.google.com/compute/docs/quickstart) only. The
+given [GCE metadata](https://cloud.google.com/compute/docs/metadata) key will be
+polled for a list of instances to open in `-dir`. The metadata key is relative
+from `computeMetadata/v1/`. The format for the value is the same as the
+'instances' flag. A hanging-poll strategy is used, meaning that changes to the
+metadata value will be reflected in the `-dir` even while the proxy is running.
+When an instance is removed from the list the corresponding socket will be
+removed from `-dir` as well (unless it was also specified in `-instances`), but
+any existing connections to this instance will NOT be terminated.
+
+Note: `-instances` and `-instances_metadata` may be used at the same time but
+are not compatible with the `-fuse` flag.
+
+* `-ip_address_types=PUBLIC,PRIVATE`
+
+A comma-delimited list of preferred IP types for connecting to an instance. For
+example, setting this to PRIVATE will force the proxy to connect to instances
+using an instance's associated private IP. Defaults to `PUBLIC,PRIVATE`
+
+* `-term_timeout=30s`
+
+How long to wait for connections to close before shutting down the proxy.
+Defaults to 0.
+
+* `-skip_failed_instance_config`
+
+Setting this flag will allow you to prevent the proxy from terminating when some
+instance configurations could not be parsed and/or are unavailable.
```

Comment:
```suggestion
Setting this flag will allow you to prevent the proxy from terminating if any errors
occur during instance configuration. Please note that this may mean some instances
may fail to set up correctly while others work if the proxy restarts..
```